src/Features/Core/Portable/Completion/CompletionChange.cs

```diff
@@ -41,9 +41,6 @@ private CompletionChange(ImmutableArray<TextChange> textChanges, int? newPositio
         private CompletionChange(TextChange textChange, int? newPosition, bool includesCommitCharacter)
         {
             TextChange = textChange;
-#pragma warning disable CS0618 // Type or member is obsolete
-            TextChanges = ImmutableArray.Create(textChange);
```

Comment:
Not setting this break binary compat (behavior-wise), for people who haven't recompiled yet.